esp-hal/src/rmt.rs

```diff
@@ -927,6 +931,7 @@ where
     Dm: crate::DriverMode,
     Raw: ChannelInternal,
 {
+    #[cfg_attr(place_rmt_driver_in_ram, ram)]
```

Comment:
Not sure about this one: This is _not_ on the time-critical path (not called during transactions, and if transactions should happen as quickly as possible one after another, the channel should be re-used rather than dropped), might not be worth forcing this to RAM.